src/main/kotlin/io/titandata/titan/providers/local/RemoteLog.kt

```diff
@@ -59,6 +60,22 @@ class RemoteLog (
                         println("Email: ${commit.properties["email"]}")
                     }
                     println("Date:   ${commit.properties["timestamp"]}")
+                    if (commit.properties.containsKey("tags")) {
+                        @Suppress("UNCHECKED_CAST")
```

Comment:
Why do you use the `@Suppress` here but not in local/Log.kt?

Reply:
Just because I noticed the warning (we don't have warnings as errors turned on to keep these from creeping in). The other one got lost in the list of existing ones, will address there as well.